CodeEdit/WorkspaceView.swift

```diff
@@ -53,18 +53,19 @@ struct WorkspaceView: View {
 		LeadingSidebar(workspace: workspace, windowController: windowController)
 			.frame(minWidth: 250)
 		HSplitView {
-			ZStack {
-				WorkspaceCodeFileView(windowController: windowController,
-									  workspace: workspace)
-			}
-			.safeAreaInset(edge: .bottom) {
-				if let url = workspace.fileURL {
-					StatusBarView(workspaceURL: url)
-				}
-			}
-			InspectorSidebar(workspace: workspace, windowController: windowController)
-				.frame(minWidth: 250, maxWidth: 250, maxHeight: .infinity)
-		}
+            ZStack {
+                WorkspaceCodeFileView(windowController: windowController, workspace: workspace)
+                    .frame(minWidth: 150, maxHeight: .infinity)
```

Comment:
change `minWidth` to 500 so the status bar will not clip

Reply:
done 👍